src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx

```diff
@@ -86,6 +86,61 @@ Decisions determine how incident intelligence correlates issues together. New Re
 3. To see examples of incidents the decision correlated, click the **Recent correlations** tab.
 4. You have the option to enable or disable these global decisions.
 
+## Configure sources [#configure-sources]
+
+Before setting up custom decisions, determine your incident sources. Sources are your data inputs.
+
+You can get data from any of the following sources:
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="configure-source-nr-alerts"
+    title="Alerts"
+  >
+    By integrating incident intelligence with your alerts violations, you can get context and correlations from what you're monitoring.
+    To get data from alerts:
+
+    1. From **[one.newrelic.com](https://one.newrelic.com)**, click **Alerts**.
+    2. On the left under **incident intelligence**, click **Sources** and then click **Alerts**.
+    3. Select the policies you want to connect to applied intelligence, and click **Connect**.
+
+       You can add additional alerts policies or remove policies you've already connected in **Sources > Alerts**.
+
+       <Callout variant="tip">
+         Adding alerts as a source will not affect your current configuration or notifications.
+       </Callout>
+  </Collapser>
+
+  <Collapser
+    className="freq-link"
+    id="configure-aporia"
+    title="Aporia (MLOps)"
+  >
+    By integrating incident intelligence with your Aporia machine-learning models, you can monitor your machine learning model performance. To configure our Aporia integration, see our [docs](/docs/integrations/mlops-integrations/aporia-mlops-integration/).
```

Comment:
For see our [docs], I'd have a longer phrase besides 'docs' so users know roughly what kind of doc it is